Makefile

```
@@ -106,6 +106,12 @@ endif
 KUADRANT_NAMESPACE ?= kuadrant-system
 OPERATOR_NAMESPACE ?= $(KUADRANT_NAMESPACE)
 
+# Kuadrant Service Account
+KUADRANT_SA_NAME ?= kuadrant-operator-controller-manager
+
+#Kuadrant Extensions
+EXTENSIONS_DIRECTORIES ?= $(shell ls -d cmd/extensions/*/)
```

Should this use the `PROJECT_PATH` var just in case

```suggestion
EXTENSIONS_DIRECTORIES ?= $(shell ls -d $(PROJECT_PATH)/cmd/extensions/*/)
```